gnss_lib_py/utils/ephemeris_downloader.py

```diff
@@ -532,6 +662,32 @@ def _valid_ephemeris_in_paths(date, possible_types, file_paths=None):
                 if os.path.split(path)[1][3:] == str(gps_week).zfill(4) + str((timetuple.tm_wday+1)%7) + ".sp3":
                     return True, path
 
+        # sp3 before Aug 13, 2017 and May 25, 2012 or later
```

Comment:
The comment may not be clear compared to the text at the start. Perhaps it could be `# sp3 between May 25, 2012 and Aug 12, 2017` to match the previous text on line 57.

Reply:
updated as suggested